cmd/template/cluster/provider/templates/aws/cluster.go

```diff
@@ -63,7 +63,7 @@ func NewClusterCRs(config ClusterCRsConfig) (ClusterCRs, error) {
 		}
 
 		if config.ControlPlaneName == "" {
-			generatedName, err := key.GenerateName(config.EnableLongNames)
+			generatedName, err := key.GenerateName(true)
```

Comment:
Same as above

Reply:
We call it with `false` when generating names for catalogs. That behaviour is older than this PR, so I thought it might be safer to keep it.

Reply:
Alright then :) That makes sense.